kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala

```diff
@@ -59,7 +59,13 @@ case class CassandraSinkSetting(keySpace: String,
                                 threadPoolSize: Int,
                                 consistencyLevel: Option[ConsistencyLevel],
                                 taskRetries: Int = CassandraConfigConstants.NBR_OF_RETIRES_DEFAULT,
-                                enableProgress: Boolean = CassandraConfigConstants.PROGRESS_COUNTER_ENABLED_DEFAULT) extends CassandraSetting
+                                enableProgress: Boolean = CassandraConfigConstants.PROGRESS_COUNTER_ENABLED_DEFAULT,
+                                deleteEnabled: Boolean = CassandraConfigConstants.DELETE_ROW_ENABLED_DEFAULT,
+                                deleteStatement: String = CassandraConfigConstants.DELETE_ROW_STATEMENT_DEFAULT,
+                                deleteStructFields: String = CassandraConfigConstants.DELETE_ROW_STRUCT_FLDS) extends CassandraSetting {
+
+  def structFlds: Seq[String] = deleteStructFields.split(",").map(s => s).toSeq
```

Comment:
Is it not simple to have this a Seq[String] as a class member?

Reply:
Done